src/config.ts

```
@@ -150,40 +154,73 @@ const commonMuteConfig = Schema.object({
         )
 })
-    .description('工具与多模态')
+    .description('工具调用回复')
     .collapse()
 
+const commonModelFeatureConfig = Schema.intersect([
+    Schema.object({
+        toolCalling: Schema.boolean()
+            .description(
+                '是否启用工具调用功能（可在[**这里**]' +
+                '(https://cooksleep.github.io/newapi-special-test)' +
```

ESLint

[error] Insert `····`

(prettier/prettier)